Makefile

```diff
@@ -1,5 +1,5 @@
 GO_REQUIRED_MIN_VERSION = 1.17
-GO_TEST_FLAGS=-v
+# GO_TEST_FLAGS=-v
```

Comment:
Just minor nag, why comment out?

Reply:
Thanks for the catch, missed in self review, I meant to remove it, so that it does not give verbose flags for unit tests written by @slaskawi 
@slaskawi shall I keep it or remove it, since now our e2e tests are not technically tests, they compile to a binary, so I do not have any need for this

Reply:
I'm OK with removing it.